src/js/components/Provenance/Catalogue/Catalogue.tsx

```diff
@@ -0,0 +1,16 @@
+import { Space } from 'antd';
+import CatalogueCard from '@/components/Provenance/Catalogue/CatalogueCard';
+import { useMetadata } from '@/features/metadata/hooks';
+
+const Catalogue = () => {
+  const { projects } = useMetadata();
+  return (
+    <Space direction="vertical" style={{ width: '100%' }}>
```

Comment:
the gap here isn't big enough visually between `CatalogueCard`s - should be the same as cards in the overview, and scale the same with screen width (like with the overview)